source/getting-started.txt

```diff
@@ -18,63 +19,151 @@ Introduction to Relational Migrator
    :depth: 1
    :class: singlecol
 
-Relational Migrator is a tool to help you migrate your relational 
-data to MongoDB. This page guides you through installing the tool 
-and running your first migration job.
+Relational Migrator is a tool to help migrate your relational data to MongoDB.
+This page provides an overview of Relational Migrator's :ref:`key features <rm-key-features>`, 
+:ref:`use cases <rm-use-cases>`, :ref:`migration scenarios <rm-migration-scenarios>`, 
+and :ref:`supported databases and versions<supported-databases>`.
 
-To get familiar with Relational Migrator capabilities, 
-features, and use cases, review the following resources:
+.. _rm-key-features:
 
-- :ref:`<supported-databases>`
--  :ref:`<rm-migration-scenarios>`
-- `Relational Migrator Learning Byte <https://learn.mongodb.com/courses/mongodb-relational-migrator>`__
+Key Features
+------------
 
-Steps
------
+Map and Model Data
+~~~~~~~~~~~~~~~~~~
 
-.. procedure::
+Relational Migrator uses entity-relationship diagrams to map your relational schema 
+to a MongoDB schema, and can generate a recommended mapping to use as a starting 
+point for your migration. For more information, see :ref:`<rm-mapping-rules>`.
 
-   .. step:: Install
+Migrate to MongoDB
+~~~~~~~~~~~~~~~~~~
 
-      You can install Relational Migrator with a
-      :ref:`single machine install <local-machine-install>` suitable 
-      for testing or a :ref:`unattended server install <unattended-server>` 
-      for production workloads.
+Relational Migrator helps you migrate data from 
+:ref:`supported relational databases <supported-databases>` to MongoDB Atlas or 
+a self-managed MongoDB deployment.
 
-   .. step:: Create a project
 
-      To start using Relational Migrator,
-      :ref:`create a project <rm-projects>`. Projects hold the 
-      settings, schema conversion logic, and table mappings for a 
-      migration. You can connect to source and destination databases with the
-      appropriate :ref:`connection strings <rm-connection-strings>` and save
-      the source and destination :ref:`database connections <rm-database-connections>`.
-      
+Generate Application Code
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. step:: Create mapping rules with suggested mappings
+Relational Migrator generates MongoDB-compatible C#, Java, and JavaScript application code
+for entity classes, persistence layers, and APIs. 
 
-      Use :ref:`suggested mappings<rm-new-rules-from-mappings>` to 
-      create initial mapping rules based on your 
-      relational schema. Mapping rules specify how data from each 
-      relational table migrates to MongoDB and how columns map to 
-      fields in MongoDB documents.
+Code generation templates support JSON output and common application frameworks like 
+Spring Data, Node, and Mongoose. For more information, see :ref:`<rm-code-generation-templates>`.
```

Comment:
[Nit]

Technically, Node is not a framework, it's a runtime environment. Not sure how best to edit this. Maybe Product can chime in?